test/integration.py

```diff
@@ -237,6 +239,63 @@ def test_no_args_shows_help(self):
         self.expectReturncode(proc, ExitStatus.USAGE_ERROR)
         self.assertIn(b"usage: ", proc.stdout)
 
+    def test_remove_mappings(self):
```

Comment:
The warning with these `.py` tests is they don't run in the CI, and we're slightly deprecating them in favour of the new framework. However this doesn't fit nicely in the new framework yet so this seems fine.